1

```diff
@@ -0,0 +1,17 @@
+wip
+
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+#
+# Date:      Wed Sep 16 15:45:47 2020 +1200
+#
+# On branch loc_block
+# Your branch and 'origin/master' have diverged,
+# and have 1 and 14 different commits each, respectively.
+#   (use "git pull" to merge the remote branch into yours)
+#
+# Changes to be committed:
+#	modified:   CRM/Event/Form/ManageEvent/Location.php
+#	new file:   tests/phpunit/CRM/Event/Form/ManageEvent/LocationTest.php
+#	modified:   tests/phpunit/CiviTest/CiviUnitTestCase.php
+#
```

Comment:
@eileenmcnaughton this file appears to have been added by mistake?

Reply:
opps - fixed